README.md

```diff
@@ -75,6 +75,24 @@ a.read_msl_data()
 a.calculate()
 ```
 
+#### Working with Zarr and Object Storage
+The `asli` package also supports Zarr data import from s3 storage through the python interface. The method remains the same, but you will need to install the [s3] optional dependencies.
+
+```sh
+pip install pip install git+https://github.com/scotthosking/amundsen-sea-low-index[s3]
+```
+
+Additionally you will need to provide the location of your s3 config file, to the `ASLICalculator` class:
```

Comment:
Do we need to show an example config file? Or is this common knowledge?

Reply:
Good point, will show an example and possibly refer to JASMIN docs specifically.